### ISSUE TITLE
Consider having one master document for all

### ISSUE THREAD

**Issue by rkrug** [ipbesdata/IPBES_Ontology#2]
At the moment it seems that the `.ttl` and the `.md` are both created manually. It would be good if there is one source document (now the `.md`, possibly better a `.qmd` file which can also contain code to be executed) which, when compiled either manually or a GitHub action (or makefile?) creates the `.html`, `.pdf` and the `.ttl` file.

**Comment by rkrug** [comment]
The solution could be to have all info up to and including the section 4 header (`4. Classes and Attributes`) in the `qmd` file, and the rest is generated by using the info in the `.ttl` file? This could be done by using some R code which reads it in and outputs it formatted as needed for the table.

**Comment by ipbesdata** [comment]
Yes, Maral created the .ttl file (not manually), and I created the .md file (manually). 

The .ttl file includes all the chapters and authors list.

**Comment by rkrug** [comment]
How did she create the file? There must have been a manual process somewhere?

**Comment by niamir** [comment]
@MaralDadvar
